pytorch_transformers/tokenization_utils.py

```diff
@@ -174,8 +174,11 @@ def _from_pretrained(cls, pretrained_model_name_or_path, cache_dir=None, *inputs
             for file_id, file_name in all_vocab_files_names.items():
                 if os.path.isdir(pretrained_model_name_or_path):
                     full_file_name = os.path.join(pretrained_model_name_or_path, file_name)
-                else:
-                    full_file_name = pretrained_model_name_or_path
+                elif os.path.isfile(pretrained_model_name_or_path):
```

Comment:
This looks a bit strange.
Here we are changing the filename part of a direct path to an existing file that was provided by the user? In which case would you like to do that?
Sound like something not natural for the user.